docs/usage/types.md

```diff
@@ -707,3 +708,28 @@ Similar validation could be achieved using [`constr(regex=...)`](#constrained-ty
 formatted with a space, the schema would just include the full pattern and the returned value would be a vanilla string.
 
 See [Schema](schema.md) for more details on how the model's schema is generated.
+
+## Allow Arbitrary Types
```

Comment:
can we link to here from the line on this in `model_config.md`.

Reply:
Or this could be moved to `model_config.md`, then a link from "Generic Classes as Types"?

Reply:
(if so you should also change the example name to keep it consistent)

Reply:
I'll add a link to here from `model_config.md` if that's okay.

Otherwise, on the first read of the docs, I feel it could give the impression that arbitrary types are allowed only/mainly if they are generics (even if we add a small link to the other section). That would make it a bit less explicit that `allow_arbitrary_types` is already very useful and powerful without having to understand and deal with generics.